api/renter.go

```diff
@@ -29,7 +31,6 @@ type FileInfo struct {
 	UploadProgress float32
```

Comment:
The important difference is that `UploadProgress` indicates the *total* progress, including redundancy and encryption overhead. So a `NumBytesUploaded` field wouldn't be very helpful. Even if `NumBytesUploaded > Filesize`, the file may not be available yet.